CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## [1.0.16] - 2024-02-24
+
+### Fixed
+
+-   Typo on `README.md` file
+-   Typo on `src/components/button/button.tsx` file
+-   Typo on `src/utils/formatterUtils/formatterUtilsDefinition.ts` file
```

Comment:
There's a conflict with the `CHANGELOG` file, also just one entry for the fix would be enough:
```
-   Typos on documentation and comments
```

Reply:
Fixed.